### PR TITLE
Add missing Modelsim error

### PR DESCRIPTION
**Pull request by derrickgw**
The current parser can't detect suppressible errors, but it should.

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
Is it a Hacktoberfest pull request just in case? Just to label it properly on our side. See [Hacktoberfest in Jenkins](https://jenkins.io/events/hacktoberfest). If you do not participate in it, please consider doing so. Your pull requests are much appreciated 👍

**Comment by uhafner** [comment]
Thanks for the PR. Can you please check the Codacy analysis results: https://app.codacy.com/manual/uhafner/analysis-model/pullRequest?prid=4323533

File: src/main/java/edu/hm/hafner/analysis/parser/MentorParser.java [hunk 1]
```diff
@@ -32,12 +32,17 @@
 
     /**
      * The first capture group matches the filename and line number.
```
Comment (uhafner) [inline src/main/java/edu/hm/hafner/analysis/parser/MentorParser.java]: and the second group matches the line number

**Comment by uhafner** [comment]
Thanks!

**Pull request merged**